HyperV/templates/policy.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
We should use this from the nova build folder rather than a hardcoded file here.